src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -238,7 +238,94 @@ public Task Get()
                 }
             }
         }
-        
+
+        [RavenAction("/admin/config-expiration-bundle", "POST", "/admin/config-expiration-bundle?name={databaseName:string}")]
```

Comment:
As we spoke i'll keep it in a server wide handler because we need to wake up the database